### DISCUSSION TITLE
Feature request: child and direct child variants

### DISCUSSION THREAD

**Discussion by PositiveUser** [tailwindlabs/tailwindcss#8108] [General]
Check this [Tailwind Play](https://play.tailwindcss.com/aMVTiJKIyK)

This allows me to not use the same classes for each child and change classes just once for every child.

**Comment by ausminternet** [comment]
I would love to see this build in tailwind!

I suggest to change your code a little bit, so that it affects only the first *direct* child: `addVariant('child-first', '& > :first-child')`

**Reply by PositiveUser** [reply]
Thanks for suggesting

[Updated code](https://play.tailwindcss.com/aMVTiJKIyK)

**Reply by richwebmedia** [reply]
This is great! how would one go about using this method to target 5 elements deep?

<div>
  <form>
    <span>
      <input>

**Reply by PositiveUser** [reply]
`direct-child:direct-child:direct-child:direct-child:direct-child:*`
